apps/web/lib/contract.ts/contract.client.ts

```diff
@@ -0,0 +1,20 @@
+import { ethers } from "ethers";
+import { FEEDBACK_FACTORY_CONTRACT_ADDRESS } from "./config";
+import { abi } from "packages/form-XChange/build/contracts/FeedbackFormFactory.json";
+import { FeedbackFormInstance } from "packages/form-XChange/types/truffle-contracts";
+
+/*
+ * Relies on window.ethereum being present (metamask installed and client side). use condionally
+ */
+export const getFormContractClientInstance = () => {
```

Comment:
This function should receive the address as a parameter and have `FEEDBACK_FACTORY_CONTRACT_ADDRESS` as default.